plugin/src/main/java/com/denizenscript/denizen/objects/AreaContainmentObject.java

```diff
@@ -117,6 +123,9 @@ static <T extends AreaContainmentObject> void registerTags(Class<T> type,  Objec
         // @returns ListTag(PlayerTag)
         // @description
         // Gets a list of all players currently within the area.
+        // @Example
+        // # Narrates a list of players' names that are within the area separated by a comma and a space.

```

Comment:
Give an example of what you'd expect this to actually output as. Like `bob, joe, steve`